### ISSUE TITLE
Unable to parse some "*.info" files 

### ISSUE THREAD

**Issue by pandom79** [pghvlaans/sbotools#19]
* Command entered:
sbocheck

* Running version of `sbotools`:
sbotools version 3.2.1

* Output of `cat /etc/slackware-version`:
Slackware 15.0

* Output of `sboconfig --list`:
sboconfig -b|--build-ignore:
    BUILD_IGNORE=FALSE
sboconfig -C|--classic:
    CLASSIC=FALSE
sboconfig -d|--distclean:
    DISTCLEAN=FALSE
sboconfig -B|--branch:
    GIT_BRANCH=FALSE
sboconfig -g|--gpg-verify:
    GPG_VERIFY=FALSE
sboconfig -j|--jobs:
    JOBS=8
sboconfig -o|--local-overrides:
    LOCAL_OVERRIDES=/home/domenico/Modelli/Slackbuilds
sboconfig -c|--noclean:
    NOCLEAN=FALSE
sboconfig -p|--pkg-dir:
    PKG_DIR=FALSE
sboconfig -r|--repo:
    REPO=git://git.slackbuilds.org/slackbuilds.git
sboconfig -R|--rsync:
    RSYNC_DEFAULT=FALSE
sboconfig -s|--sbo-home:
    SBO_HOME=/usr/sbo
sboconfig -V|--slackware-version:
    SLACKWARE_VERSION=FALSE

* Description:
When i run "sbocheck", i get the following error:
```
Updating SlackBuilds tree...
HEAD ora si trova a 07c737e354 Public www update: Sat Dec 28 01:30:01 UTC 2024.
Si è già su '15.0'
Il tuo branch è aggiornato rispetto a 'origin/15.0'.
Già aggiornato.

Checking for updated SlackBuilds...A fatal script error has occurred:
Error when parsing file python3-evdev.info.
Exiting.
```
  It seems that it isn't able to parse "*.info" file only for some packages.

**Comment by pghvlaans** [comment]
@pandom79 Thank you for taking the time to make an Issue. There was a report about this yesterday as well in #18 and some new commits have gone in that appear to have fixed the problem. The easiest way to install the latest commit from the master branch is to clone [sbotools-git-slackbuild](https://github.com/pghvlaans/sbotools-git-slackbuild) and use the files there. I'll have a new version out for this coming weekend.  

Basically, some `info` files are missing a quotation mark at the moment. As of the latest release (and also in sbotools-2.7), the script errors out. The fix was to repair missing quotes and backslashes, bad lines, etc. before parsing. Please let me know if the master branch code works for you.

**Comment by pandom79** [comment]
Yes, it works.
